changelog/fragments/allow-v-in-version.yaml

```diff
@@ -0,0 +1,18 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      Allows the `v` char as a prefix in version string. The `v` char in 
+      version string, is extremely common, for example the `git describe`
+      command produces an output like this: `v0.2.0-13-ga74dac1`.
```

Comment:
Could you expand on (a) which versions are allowed to have the `v` prefix and (b) what commands will translate those versions to non-`v`-prefixed versions and (c) which output artifacts will drop the `v` prefix and which won't?

IIRC there's a handful of places where versions are specified by users and a separate handful where versions are written/pushed by the `operator-sdk` commands or the project Makefile.

Reply:
Hope the new description is more informative. Please, re-evaluate @joelanford.

Reply:
@joelanford please take a look once again. Thanks!